### PR TITLE
Bump develop to 3.12.0-develop

### PR DESCRIPTION
**Pull request by Griffin-Sullivan**
Removed translations from this PR and now it's for develop branch version bump.

### CODE REVIEW

File: locale/foreman.pot
Comment (ekohl) [inline locale/foreman.pot:1]: Somehow this appears to be skipping all of `webpack/assets`. I wonder if that was some change in a dependency or something in your environment.

Comment (Griffin-Sullivan) [reply locale/foreman.pot:1]: Do you want to run the script yourself and see what you get?

Comment (ekohl) [reply locale/foreman.pot:1]: I see `extract-strings` stops with:
```
MALFORMED: "Invalid value '%s' for jwt_expiration. The value must be between %s and %s. 0 "
Malformed strings found: 1
```

This was introduced in 3.10 so we should have found this earlier.

**Comment by ekohl** [comment]
How did you even generate this? If I run `make -C locale tx-update` I get something very else. But it's also very unpredictable. I think https://github.com/theforeman/foreman/pull/10177 is how it should be.

**Comment by Griffin-Sullivan** [comment]
Well now I'm just confused...

File: locale/foreman.pot
Comment (Griffin-Sullivan) [reply locale/foreman.pot:1]: Yeah I get the same issue

**Pull request merged**